src/bindings/python/src/openvino/frontend/pytorch/utils.py

```diff
@@ -0,0 +1,146 @@
+    """
+    target_atoms = target.split('.')
+    attr_itr = self_module
+    for i, atom in enumerate(target_atoms):
+        if not hasattr(attr_itr, atom):
+            raise RuntimeError(f"Node referenced nonexistent target {'.'.join(target_atoms[:i])}")
+        attr_itr = getattr(attr_itr, atom)
+    return attr_itr
+
+
+def get_type_from_py_type(value):
+    if isinstance(value, float):
+        return OVType.f32
+    if isinstance(value, bool):
+        return OVType.boolean
+    if isinstance(value, int):
+        # Python int is 64 bit, but we will convert it to int32 except cases when it can't fit in 32 bits
+        if torch.iinfo(torch.int).min <= value <= torch.iinfo(torch.int).max:
+            return OVType.i32
+        return OVType.i64
+    return OVType.dynamic
+
+
+def ivalue_to_constant(ivalue):
+    ov_type = get_type_from_py_type(ivalue)
+    if ov_type.is_static():
+        return op.Constant(ov_type, Shape([]), [ivalue]).outputs()
+
+    if isinstance(ivalue, (list, tuple)):
+        assert len(ivalue) > 0, "Can't deduce type for empty list"
+        ov_type = get_type_from_py_type(ivalue[0])
+        assert ov_type.is_static(), "Can't deduce type for list"
+        return op.Constant(ov_type, Shape([len(ivalue)]), ivalue).outputs()
+
+    if isinstance(ivalue, torch.Tensor):
+        ivalue = ivalue.to(memory_format=torch.contiguous_format)
+        if ivalue.dtype == torch.bfloat16:
+            # reinterpret bfloat16 data as float16 to allow conversion to numpy
+            ivalue = ivalue.view(torch.float16)
+            narr = ivalue.numpy(force=True)
+            if not narr.flags['C_CONTIGUOUS']:
+                narr = np.ascontiguousarray(narr)
+            # TODO: this tensor doesn't share memory with initial tensor
+            tensor = Tensor(narr, ivalue.shape, OVType.bf16)
+            ov_const = op.Constant(tensor, shared_memory=True)
+        else:
+            narr = ivalue.numpy(force=True)
+            if not narr.flags['C_CONTIGUOUS']:
+                narr = np.ascontiguousarray(narr)
+            ov_const = op.Constant(narr, shared_memory=True)
+        return ov_const.outputs()
+    return None
+
+def get_value_from_getattr(getattr_node, self_module):
+    assert getattr_node.kind() == "prim::GetAttr", "Got node of kind not equal to prim::GetAttr"
+    # GetAttr nodes can be nested
+    stack = []
+    while getattr_node.kind() == "prim::GetAttr":
+        stack.append(getattr_node)
+        inputs = list(getattr_node.inputs())
+        if len(inputs) == 0:
+            break
+        getattr_node = inputs[0].node()
+    module = self_module
+    while len(stack) > 0:
+        node = stack.pop()
+        attr_name = node.s("name")
+        assert hasattr(module, attr_name), f"No attribute with name \"{attr_name}\" found in module."
+        module = getattr(module, attr_name)
+    return module
+
+
+pt_to_ov_type_map = {
+    "float": OVType.f32,
+    "int": OVType.i32,
+    float: OVType.f32,
```

Comment:
You don't need actual types here, you need to just use `str(type)` to search this map.

Reply:
Updated accordingly.